torch/csrc/jit/codegen/cuda/executor_utils.cpp

```diff
@@ -523,7 +525,10 @@ kir::ExpressionEvaluator bindKernelInputs(
       kernel->inputs().size() == aten_inputs.size(),
       "Something went wrong configuring launch. Inputs no longer match.");
 
-  kir::ExpressionEvaluator expr_eval;
+  auto mapping = glfdc::ReusedExprMapping::create_lazy_mapping(
+      kernel->summary().expr_reuses);
+  std::unique_ptr<glfdc::EvalState> es{new glfdc::EvalState{mapping}};
+  kir::ExpressionEvaluator expr_eval{std::move(es)};
```

Comment:
Needs better API

Reply:
Expression reuses bitmap was moved from ExprBuilder into ExprDAG; thus we don't need to deal with it and creating EvalState is simpler. I've added helper createEvalStateForDAG() that handles it right now.